lib/app/modules/home/widget/recommended.dart

```diff
@@ -0,0 +1,92 @@
+  const RecommendedWidget({
+    Key? key,
+  }) : super(key: key);
+
+  @override
+  Widget build(BuildContext context) {
+    final controller = Get.find<HomeController>();
+    return Container(
+        width: appService.swidth,
+        height: appService.sheight * .26,
+        child: Column(
+          mainAxisAlignment: MainAxisAlignment.start,
+          crossAxisAlignment: CrossAxisAlignment.start,
+          children: [
+            HeaderText(
+              Strings.recommended,
+              textColor: AppColors.textColor,
+            ),
+            Expanded(
+              child: ListView(
+                shrinkWrap: true,
+                scrollDirection: Axis.horizontal,
+                children: controller.recommendedUserList
+                    .map((e) => Container(
+                          height: appService.sheight * .14,
+                          width: appService.swidth * .4,
+                          padding: EdgeInsets.all(Constants.defaultmargin / 2),
+                          margin: EdgeInsets.all(Constants.defaultmargin / 2),
+                          child: Column(
+                            mainAxisAlignment: MainAxisAlignment.start,
+                            crossAxisAlignment: CrossAxisAlignment.start,
+                            children: [
+                              Expanded(
+                                child: Container(
+                                  child: Image.asset(e.image),
+                                  decoration: BoxDecoration(
+                                      color: e.background,
+                                      borderRadius: BorderRadius.circular(
+                                          Constants.defaultmargin)),
+                                ),
+                              ),
+                              HeightWidget(h: .001),
+                              HeaderText(
+                                e.title,
+                                textColor: AppColors.textColor,
+                                fontSize: appService.swidth * .05,
+                              ),
+                              HeightWidget(h: .001),
+                              Row(
+                                mainAxisAlignment: MainAxisAlignment.start,
+                                children: [
+                                  NormalText(
+                                    e.subtitle.toUpperCase(),
+                                    fontSize: appService.swidth * .031,
+                                    color: AppColors.pRIMARYCOLORLIGHT,
+                                  ),
+                                  WidthWidget(h: .01),
+                                  CircleAvatar(
+                                      radius: Constants.defaultRadus / 9,
+                                      backgroundColor:
+                                          AppColors.pRIMARYCOLORLIGHT),
+                                  WidthWidget(h: .008),
```

Comment:
for specific named widgets, you should simply pass the value instead of specifying the attribute. Also, what is the height attribute used for in width widget?

Reply:
I already create width and height fix size so it just take a simple value and auto convert to respective height reduce long and full code.

Reply:
it should just be WidthWidget(.008),

Reply:
done

Reply:
done